### PR TITLE
Keep provider and factory available at old FQN until next major update.

### PR DESCRIPTION
**Pull request by arjantijms**
Signed-off-by: arjantijms <arjan.tijms@gmail.com>

### CODE REVIEW

File: appserver/security/core-ee/osgi.bundle [hunk 1]
```diff
@@ -25,9 +25,9 @@
                         com.sun.enterprise.security.jmac; \
                         com.sun.enterprise.security.jmac.callback; \
                         com.sun.enterprise.security.jmac.config; \
- 			com.sun.enterprise.security.auth.realm.pam; \
+ 			            com.sun.enterprise.security.auth.realm.pam; \
```
Comment (dmatej) [inline appserver/security/core-ee/osgi.bundle:28]: Formatting?

File: appserver/security/core-ee/src/main/java/com/sun/enterprise/security/jacc/provider/SimplePolicyConfigurationFactory.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/*
```
Comment (dmatej) [inline appserver/security/core-ee/src/main/java/com/sun/enterprise/security/jacc/provider/SimplePolicyConfigurationFactory.java:1]: Is it the correct license header if it is a new file?

File: appserver/security/core-ee/osgi.bundle (hunk 1 above)
Comment (arjantijms) [reply appserver/security/core-ee/osgi.bundle:28]: They were there before, but let's make them consistent

File: appserver/security/core-ee/src/main/java/com/sun/enterprise/security/jacc/provider/SimplePolicyConfigurationFactory.java (hunk 2 above)
Comment (arjantijms) [reply appserver/security/core-ee/src/main/java/com/sun/enterprise/security/jacc/provider/SimplePolicyConfigurationFactory.java:1]: I updated it, thx ;)

**Review by dmatej — APPROVED** [review]

**Pull request merged**